### DISCUSSION TITLE
budibase db not exporting all columns from a table

### DISCUSSION THREAD

**Discussion by Whizzie99** [Budibase/budibase#10013] [Help] [answered]
I'm using budibase db as my source and when I try to export data from my database table as csv, it doesn't export all columns. 
please find images of the database table and exported data below
<img width="1139" alt="Screenshot 2023-03-14 at 09 47 07" src="https://user-images.githubusercontent.com/47413415/224945992-f5d3de2c-b182-4f93-b596-33bc2ac525d1.png">
<img width="1440" alt="Screenshot 2023-03-14 at 09 47 28" src="https://user-images.githubusercontent.com/47413415/224946001-c5802fd7-f60d-4fbc-9f8d-b9e7792d93b8.png">

**Comment by melohagan** [comment, marked as answer]
Hey @Whizzie99 

I'm assuming all of the columns that are not exported are relationship fields? They are not exported because when importing there is no guarantee that the related data would exist. 

E.g. if you exported from one app and imported into another all of the relationship link ids would be invalid, so there's no point exporting the relationships.

What you can do however is use **static** formulas to pull a field value from a relationship and then you can export that:

![Screenshot 2023-03-15 at 14 46 02](https://user-images.githubusercontent.com/101575380/225345946-02cd8c61-2c82-4d86-a3cd-ee3c3aeafb4c.png)

![Screenshot 2023-03-15 at 14 45 13](https://user-images.githubusercontent.com/101575380/225345729-123eea89-bae4-4d55-8024-164ea91945f0.png)

![Screenshot 2023-03-15 at 14 46 26](https://user-images.githubusercontent.com/101575380/225346053-0abeefaf-d947-4dfe-b31b-8a537543eb4a.png)

![Screenshot 2023-03-15 at 14 44 44](https://user-images.githubusercontent.com/101575380/225345591-57fced1a-89ab-46e2-a9d1-d7fecd61c976.png)

### Result of export
![Screenshot 2023-03-15 at 14 47 15](https://user-images.githubusercontent.com/101575380/225346281-deb8eaf5-92d7-4223-9311-9203109e5470.png)

> The Quote Price from the Job relationship is included in the export, however the relationship field 'Job' is not included.
